### PR TITLE
Add initial test suite

### LINKED ISSUE #63: Add a test suite
We need some kind of test suite to verify builds. Currently we just test that the js code finish successfully, but the result can be totally broken and it will pass.

### PR DESCRIPTION
**Pull request by chearon**
This isn't much, but is hopefully a good starting point.

Fixes #63

### CODE REVIEW

File: test/index.js [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+  if (this.blob) this.blob.destroy();
+  if (this.face) this.face.destroy();
+  if (this.font) this.font.destroy();
+  if (this.buffer) this.buffer.destroy();
+  this.blob = this.face = this.font = this.buffer = null;
+});
+
+describe('Face', function () {
+  it('collectUnicodes reflects codepoints supported by the font', function () {
+    this.blob = hb.createBlob(fs.readFileSync(path.join(__dirname, 'fonts/noto/NotoSans-Regular.ttf')));
```
Comment (khaledhosny) [inline test/index.js:20]: Can we reuse the fonts that are currently in `examples/` instead of adding new ones?

**Comment by chearon** [comment]
Sorry, I can still update the tests to use those fonts if you want! I hadn't seen that Mada was already variable width, and the rest of the tests should work with Roboto with some of the numbers changed.

**Comment by khaledhosny** [comment]
It is probably easier to update the examples to use the test fonts and delete the other files.

**Comment by khaledhosny** [comment]
> It is probably easier to update the examples to use the test fonts and delete the other files.

9088420e669ae85f38e7f7a7b612ddfdf8e924b8

**Pull request merged**